src/main/java/use_cases/PostManager.java

```diff
@@ -13,11 +13,28 @@
     private ArrayList<Post> posts;
     private DatabaseManager databaseManager = new DatabaseManager();
 
+    /**
+     * Constructor from the database
+     */
     public PostManager(){
         Post[] allPosts = this.databaseManager.getAllPosts();
         this.posts = new ArrayList<>(Arrays.asList(allPosts));
     }
 
+    /**
+     * Constructor given an array of posts
+     */
+    public PostManager(ArrayList<Post> posts){
+        this.posts = posts;
+    }
+
+    /**
+     * Private function in order to interacts with the posts
```

Comment:
I think we could make this more descriptive, like "Likes or comments on a given Post". If we are using the @param in the comment, I think we should add descriptions next to each param, like "@param targetPost The Post we are performing the action on"